Source/WebCore/rendering/style/TextSizeAdjustment.h

```diff
@@ -28,24 +28,34 @@ namespace WebCore {
 
 class RenderStyle;
 
-enum TextSizeAdjustmentType { AutoTextSizeAdjustment = -1, NoTextSizeAdjustment = -2 };
+enum class TextSizeAdjustmentType : bool { None, Auto };
```

Comment:
I prefer that we nest this inside the `TextSizeAdjustment` class as the name `Type`, and then refer to it as `TextSizeAdjustment::Type`.

Reply:
Done!